coordinator-external/src/main/scala/com/socrata/datacoordinator/external/DataCoordinatorError.scala

```diff
@@ -0,0 +1,110 @@
+package com.socrata.datacoordinator.external
+
+sealed class DataCoordinatorError
```

Comment:
Good question, I have thought about this, but it seems that the errors are designed in a way so that they can be added upon as needed in a general way. I would definitely like to change it as you described, but would take some significant time to get us there. I would say this is a good start to that alternative you mentioned.